sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -217,12 +219,18 @@ await foreach (IncomingRelationship incomingRelationship in incomingRelationship
                     }
 
                     // Now the digital twin should be safe to delete
-                    string digitalTwinId = twin.Key;
 
                     #region Snippet:DigitalTwinsSampleDeleteTwin
 
-                    Response deleteDigitalTwinResponse = await client.DeleteDigitalTwinAsync(digitalTwinId);
-                    Console.WriteLine($"Deleted digital twin with Id {digitalTwinId}. Response Status: {deleteDigitalTwinResponse.Status}");
+                    try
+                    {
+                        Response deleteDigitalTwinResponse = await client.DeleteModelAsync(digitalTwinId);
+                        Console.WriteLine($"Deleted digital twin with Id {digitalTwinId}. Response Status: {deleteDigitalTwinResponse.Status}");
+                    }
+                    catch (Exception ex)
```

Comment:
Shouldn't we just catch RequestFailedException ?

Reply:
yes, good point. I'll update it.